### ISSUE TITLE
[BUG] Error in Windows version when clicking 2 times any icon on the left menu

### ISSUE THREAD

**Issue by lucapellegrino70** [TiagoSilvaPereira/vemto-issues#4]
```
NavigationDuplicated: Avoided redundant navigation to current location: "/projects/1/apps".
    at $e (app://./js/chunk-vendors.f2f83789.js:194:17090)
    at Be (app://./js/chunk-vendors.f2f83789.js:194:16669)
    at t.et.confirmTransition (app://./js/chunk-vendors.f2f83789.js:194:20877)
    at t.et.transitionTo (app://./js/chunk-vendors.f2f83789.js:194:20164)
    at t.push (app://./js/chunk-vendors.f2f83789.js:194:23892)
    at app://./js/chunk-vendors.f2f83789.js:194:27581
    at new Promise (<anonymous>)
    at Dt.push (app://./js/chunk-vendors.f2f83789.js:194:27544)
    at a.goTo (app://./js/project_main.483cb940.js:1:7478)
    at click (app://./js/project_main.483cb940.js:1:1723)
```

---

vemto 0.4.14
Electron 8.5.0
win32 10.0.18362
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Thanks @lucapellegrino70 I'll check it

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.4.15
